containerz/containerz.proto

```diff
@@ -225,11 +223,10 @@ message RemoveImageResponse {
 }
 
 message RemoveContainerRequest {
-  // The container image name to be removed.
-  string name = 1;
+  // The container name to be removed.
+  string instance_name = 1;
```

Comment:
We should not just change the field name her -- any textproto instances will no longer unmarshal correctly. Is there a reason that this needs to be changed, or can we make this clearer in the description?

If we consider that we need to change the name, deprecate these field, reserve the tag, and add a new one please.

Reply:
Ah good point about textprotos, I will revert this change as it is not entirely necessary.